CONTRIBUTING.md

```diff
@@ -25,7 +25,20 @@ Happy coding!
 
 ## Netlify
 
-We use [Netlify](https://www.netlify.com/) in the background to build our page:
+We use [Netlify](https://www.netlify.com/) in the background to build and deploy our pages.
 
-- Commits to master trigger a build of the live page.
-- Pull requests trigger a build of the related branch.
+### Branching
+Our branching strategy is the following:
+- `master` is the Production branch
+- `develop` is the branch for current development. Any feature starts from it and is merged back to it once approved.
```

Comment:
@ralf57, I'm unsure about this part. Wouldn't this introduce the same issues we had with release branches? Could it be an option to have a process where merged features and bug fixes are immediately merged (including dropping tagging)? While I follow your proposal in my software projects, I unsure whether the added complexity is worth it in our current setup.

@gurix, @renestalder: What are your thoughts?

Reply:
@backflip I never assume one  merge request is exactly one "feature". Let's say you split up a big merge request into 3 separate merge requests to make them easier to review, you would probably be happy to merge over develop first and then to master.

Also, you can easily cherry-pick merged features and hotfixes to master (after possible squashes) without the need to wait for a big release. As I see the document also doesn't define if you can or not merge to master.

Reply:
Good point, did not think about the splitting. I'm fine with the "two-step" process with `develop` and `master`.